ppr-ui/src/utils/date-helper.ts

```diff
@@ -52,10 +52,19 @@ export function convertDate (date: Date, includeTime: boolean, includeTz: boolea
 }
 
 export function pacificDate (date: Date | string, omitSeconds = false): string {
-  date = new Date(date.toLocaleString('en-US', { timeZone: 'America/Vancouver' }))
```

Comment:
We are also reparsing in the `createUTCDate` Function, but that's outside the scope of my current fix.